apps/docs/src/app/core/component-docs/grid-list/examples/default/grid-list-example.component.html

```diff
@@ -0,0 +1,25 @@
+<fd-grid-list>
+    <fd-grid-list-title-bar title="Products" [totalCount]="list.length"></fd-grid-list-title-bar>
```

Comment:
I think that totalCount should be a part of content into title-bar, for example, 'additional-title-item'. And developer can control when it'll show this additional item, same as control for more button.